testsuite/features/build_validation/add_custom_repositories/add_ubuntu_repositories.feature

```diff
@@ -1,6 +1,7 @@
 # Copyright 2021-2023 SUSE LLC
```

Comment:
If that's only for ubuntu18, it mighe be better to rename the filename?

Reply:
It used to be for 16.04 as well.

In any case, I plan a complete renaming of those features. We currently have a mess between "custom channels" and "common channels". I delayed it because I did not want to interfere more with Maxime's work.